rules update

mazen160 opened the pull request:
Added new rules for FPs we have seen.

To test:

```shell
./trufflehog-fp-analyzer.py --rules rules.yaml --validate-rules  # CI rules validation
./trufflehog-fp-analyzer.py --rules rules.yaml --trufflehog-json tests/fp-4.json  # Test for the observed pattern

arkadiyt commented on rules.yaml:
```diff
@@ -105,3 +105,31 @@ patterns:
+  - pattern:
+      name: docker image hash
+      type: regex
+      value: '(image:\ )chimebank/server-gsimulator(.)+{IDENTIFIED_STRING}'
```
server-gsimulator is too specific probably - can this be `chimebank/(alphanumeric name):(identified string)`?

mazen160 replied:
Yes, you're right, It seems I missed it

mazen160 replied:
Updated, thanks for pointing

arkadiyt commented on rules.yaml:
```diff
@@ -105,3 +105,31 @@ patterns:
+  - pattern:
+      name: a-z
+      type: regex
+      value: "(ABCDEFGHIJKLMNOPQRSTUVWXYZ|abcdefghijklmnopqrstuvwxyz)"
```
This seems very specific

mazen160 replied:
Yes, this particular one has been detecting a-z list as an event. The string that has been reported multiple times is `ABCDEFGHIJKLMNOPQRSTUVWXYZ` - for a list of chars for some string generation

arkadiyt commented:
Also can you please add a rule excluding ssh keys - we had a lot of alerts like this one: https://app.circleci.com/pipelines/github/1debit/server-ops/167/workflows/74db8a53-648e-4e43-89a2-1774cd64ee9c/jobs/169

arkadiyt commented on rules.yaml:
```diff
@@ -115,7 +115,7 @@ patterns:
       name: docker image hash
       type: regex
-      value: '(image:\ )chimebank/server-gsimulator(.)+{IDENTIFIED_STRING}'
+      value: '(image:\ )chimebank/(.)+{IDENTIFIED_STRING}'
```
Still needs the colon

mazen160 commented:
> Also can you please add a rule excluding ssh keys - we had a lot of alerts like this one: https://app.circleci.com/pipelines/github/1debit/server-ops/167/workflows/74db8a53-648e-4e43-89a2-1774cd64ee9c/jobs/169

Added too. Also added a test case:
./trufflehog-fp-analyzer.py --rules rules.yaml --validate-rules  && ./trufflehog-fp-analyzer.py --rules rules.yaml --trufflehog-json tests/ssh-pub-key.json

arkadiyt reviewed (approved):
lgtm
